docs/dev/ci.md

```diff
@@ -0,0 +1,42 @@
+# CI
+
+## CI Tests
+
+Atlas Kubernetes Operator testing can be divided into tow different types of tests:
+
+- **Local Tests**: which includes `unit tests`, `linting` and things like that.
+- **Cloud Tests**: which test the operator against Atlas QA cloud performing real resource provisioning actions. These include not both `integration` and `end to end` (`e2e`) tests.
+
+Both tests differ mainly in cost: **Local Tests** are cheap because they run in a few minutes and are reliable, while **Cloud Tests** take about an hour to run, can be flaky and require access to certain secrets to manage cloud infrastructure. For this reason not all PRs should always run **Cloud Tests**.
```

Comment:
```suggestion
Both tests differ mainly in cost: **Local Tests** are cheap because they don't consume remote resource while **Cloud Tests** take a longer time tun to run, can be flaky, and are consuming remote cloud infrastructure. For this reason not all PRs should run **Cloud Tests** in all cases.
```

Reply:
I see some typos there, but I get the picture.

```suggestion
Both tests differ mainly in cost: **Local Tests** are cheap because they don't consume remote resources, while **Cloud Tests** a longer time to run, can be flaky, and consume remote cloud infrastructure. For this reason not all PRs should always run **Cloud Tests**.
```